### PR TITLE
Update responses in subscribe API - Closes #1049

### LINKED ISSUE #1049: Update subscribe API documentation
Update the reference of the subscribe API of the Lisk Service documentation.

See: https://github.com/LiskHQ/lisk-service/blob/development/docs/api/websocket_subscribe_api.md

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1049

### CODE REVIEW

File: modules/ROOT/pages/references/subscribe-api.adoc [hunk 1]
```diff
@@ -56,51 +55,65 @@ connection.on('update.block', (block) => { (...) });
...
+  },
 }
 ----
 
 === update.round
 Updates about the forging delegates for the next round.
 
 .Response
-[source,js]
+[source,json]
```
Comment (MichalTuleja) [inline modules/ROOT/pages/references/subscribe-api.adoc:99]: This change makes comments illegal according to the specs. As a result they are marked red in GitHub, not sure about the other systems. I would consider keeping it JavaScript (js) or JSON with comments (jsonc) when possible.

File: modules/ROOT/pages/references/subscribe-api.adoc [hunk 2]
```diff
@@ -109,21 +122,24 @@ Updates the current forgers' list, so the current forger is in the first positio
 
 
 .Response
-[source,js]
+[source,json]
```
Comment (MichalTuleja) [inline modules/ROOT/pages/references/subscribe-api.adoc:125]: Same as above.

File: modules/ROOT/pages/references/subscribe-api.adoc [hunk 3]
```diff
@@ -132,32 +148,45 @@ Updates about transactions from the last block.
...
```
Comment (MichalTuleja) [inline modules/ROOT/pages/references/subscribe-api.adoc:151]: Same as above.

File: modules/ROOT/pages/references/subscribe-api.adoc [hunk 4]
```diff
@@ -56,51 +55,65 @@ connection.on('update.block', (block) => { (...) });
 == Available subscriptions
 
 === update.block
-Updates about a newly forged block with its all data.
+Updates about a newly forged block with all its data.
 
 .Response
-[source,js]
+[source,json]
```
Comment (MichalTuleja) [inline modules/ROOT/pages/references/subscribe-api.adoc:61]: This change makes comments illegal according to the specs. As a result they are marked red in GitHub, not sure about the other systems. I would consider keeping it JavaScript (js) or JSON with comments (jsonc) when possible.

**Review by MichalTuleja — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/references/subscribe-api.adoc (hunk 1 above)
Comment (Tschakki) [reply modules/ROOT/pages/references/subscribe-api.adoc:99]: I only saw comments in one example and exchanged them with code callouts, should be fine now!

**Review by MichalTuleja — APPROVED** [review]
LGTM 👍

**Pull request merged**